src/DeviceInterface/initFormSubmissionsApi.js

```diff
@@ -82,6 +82,10 @@ export function initFormSubmissionsApi (forms) {
         if (!entries.length) return
 
         const filledForm = [...forms.values()].find(form => form.hasValues())
+        const focusedForm = [...forms.values()].find((form) => form.hasFocus())
+        // If a form is still focused the user is still typing: do nothing
+        if (focusedForm) return
+
```

Comment:
TBH, I was tempted to drop this stuff altogether, since now we have other more reliable ways to listen for form submission. But ultimately this doesn't hurt and gives us marginally more coverage.